mailbox/event/json/pom.xml

```diff
@@ -77,6 +82,11 @@
             <artifactId>scala-library</artifactId>
             <version>${scala.version}</version>
         </dependency>
+        <dependency>
+          <groupId>org.scala-lang.modules</groupId>
+          <artifactId>scala-java8-compat_2.12</artifactId>
```

Comment:
you should use `${scala.baseVersion}` (not sure of the exact property) instead of hardcoding 2.12